src/overlays/effects/ovl_Effect_Ss_Kakera/z_eff_ss_kakera.h

```diff
@@ -11,6 +11,11 @@ typedef enum {
     /*  1 */ KAKERA_COLOR_BROWN
 } KakeraColorIndex;
 
+typedef struct {
+    /* 0x0 */ u8 lod;
```

Comment:
What does `lod` mean?

Reply:
Level of detail; it's passed as the level of detail factor to gDPSetPrimColor.